cli/src/config/templates.toml

```diff
@@ -115,10 +115,9 @@ label(if(overridden, "overridden"),
 '''
 
 builtin_draft_commit_description = '''
-concat(
+join("\n",
```

Comment:
nit: same here. use `concat(line ++ "\n", ...)`.

Reply:
Seems like here is the perfect place to use `join()`, since there are newlines between each of these but not at the end. Or do you mean I should add a `++ "\n"` after the `surround()`?